frontend_lib/src/tinymceAutoCompleteHelper.js

```diff
@@ -93,36 +94,32 @@ export const tinymceAutoCompleteHandleKeyDown = (event, setState, isAutoComplete
 }
 
 export const tinymceAutoCompleteHandleClickItem = (item, setState) => {
-  tinymce.activeEditor.focus()
-  const selection = tinymce.activeEditor.dom.select('span#autocomplete')[0]
-  tinymce.activeEditor.selection.select(tinymce.activeEditor.dom.select('span#autocomplete')[0])
-  tinymce.activeEditor.execCommand('mceInsertContent', false, `${item.mention}&nbsp;`)
-  tinymce.activeEditor.dom.remove(selection)
-
-  setState({ isAutoCompleteActivated: false })
-}
-
-export const tinymceAutoCompleteHandleSelectionChange = (selectionId, setState, isAutoCompleteActivated) => {
-  if (selectionId === 'autocomplete__searchtext' || selectionId === 'autocomplete__start') {
-    if (!isAutoCompleteActivated) setState({ isAutoCompleteActivated: true })
+  if (!item.mention) {
+    console.log('BUG: this member does not have a username')
```

Comment:
This console log is on purpose? If yes, could you add add a flash message instead?

Reply:
It is on purpose, and not supposed to happen (thanks to the filter, that filters out). I don't think we should bother the user with a message caused by a bug. It is intended to the developers who will debug a potential issue.
Can I leave it like this or should I still show a flash message?

It's a bit like a "should never happen" message.